packages/playground/website/src/main.tsx

```diff
@@ -87,6 +88,7 @@ function Main() {
 
 	return (
 		<PlaygroundContext.Provider value={{ storage }}>
+			<ErrorReportModal />
```

Comment:
Follow-up idea – add a "Report an error" button to trigger the modal manually.

Reply:
Done [b2dfafc](https://github.com/WordPress/wordpress-playground/pull/1102/commits/b2dfafc9059717c583419dba70e4b936b46de837)